### DISCUSSION TITLE
Packer Config Error for env.keys.themeId

### DISCUSSION THREAD

**Discussion by fatihturan** [hayes0724/shopify-packer#83] [Q&A]
Hi,

When I try to start Packer with `packer start` command I see an error with `Error: [packer-config]: A value has not been defined for the key ‘env.keys.themeId’`. Where should I define the themeId value?

![image](https://user-images.githubusercontent.com/26871/115841500-878f6900-a425-11eb-9d23-d513739262b7.png)

**Comment by mrpacman101** [comment]
@fatihturan 
Hi, can you confirm you have filled out this file? 
<img width="513" alt="Bildschirmfoto 2021-04-26 um 22 00 58" src="https://user-images.githubusercontent.com/2622832/116143023-e3cee300-a6da-11eb-9460-3fdc9345a256.png">

When you use packer start - it will use "development" env by default.

Also i spotted Error: 
This is a live theme! 

I am not sure if this flag works with `packer start --allow-live`

Usually i don't use published live theme to edit, also on my development stores.

**Reply by fatihturan** [reply]
I filled the `packer.env.json` with private Shopify app id and password like this:

![Image 2021-04-30 at 2 57 52 PM](https://user-images.githubusercontent.com/26871/116692126-8ed5ea00-a9c4-11eb-8351-69e9909ba49a.jpg)

By the way I tried the Packer with different computer on Node 14.15.2 and its give error related with Babel:

![image](https://user-images.githubusercontent.com/26871/116692590-42d77500-a9c5-11eb-9fbb-43a19174e9ab.png)

**Comment by hayes0724** [comment]
For live themes you need to add `--allowLive` [Command Docs](https://hayes0724.github.io/shopify-packer/docs/commands#start)

or

live setting in [config](https://hayes0724.github.io/shopify-packer/docs/config/environment)

```
{
    "development": {
      "id": "74500041118",
      "password": "ebd6ce7f27aae8cdafb8111a5b887b9",
      "store": "my-store-name.myshopify.com",
      "live": "true",
      "ignore": [
        "settings_data.json"
      ]
    }
}

```

Can you confirm id has a correct theme id from themes page in Shopify? You can also check theme id's by using `packer theme:list`

Also it would help to have this info:

- Any flags used, example `packer start --env=live`
 - OS: [e.g. Windows 10, Ubuntu, OSX]
 - Node Version [e.g. 14.15.1]
 - NPM Version [e.g. 6.14.8]
 - Browser [e.g. chrome, safari]
 - Packer Version [e.g. 1.2.1]

**Comment by fatihturan** [comment]
I tried the Packer on the another MacBook Pro M1. I entered the required values into `packer.env.json` file. I am able to list themes and create a sample theme like this:

![image](https://user-images.githubusercontent.com/26871/119325701-d11fdd80-bc89-11eb-8eb5-f394e7e4b8b7.png)

Also I installed the packages with `npm install` command. But when I use the `packer start` I get the following errors:

![image](https://user-images.githubusercontent.com/26871/119326080-3378de00-bc8a-11eb-8fda-7dc658651680.png)

System Info: 
* macOS Big Sur 11.3.1 / MacBook Pro M1
* Node 16.2.0
* NPM 7.14.0
* Safari 14.1
* Packer v2.2.0

**Reply by mrpacman101** [reply]
@fatihturan I know it may not be related to this particular issue but i used npm before a lot - tho since latest MacOS updates i always got issues specifically on MacOS. 
Since then i switched to "yarn" and cannot complain at all, it is working like a charm no issues at all. 
Maybe worth a try.

**Reply by fatihturan** [reply]
I switched to Yarn. Removed the `node_modules` folder. And re-installed with `yarn install` command like below. But unfortunately, I still got the same errors:

![image](https://user-images.githubusercontent.com/26871/119616780-a9f01a00-be09-11eb-92c0-813917bda7de.png)

Is it related to that compatibility issue on M1 architecture? Especially for fibers package?
